src/Data/Vec/Functional/Relation/Unary/All/Properties.agda

```diff
@@ -21,79 +22,72 @@ open import Relation.Unary
 private
   variable
     a b c p q r ℓ : Level
-    A : Set a
-    B : Set b
-    C : Set c
+    A B C : Set a
```

Comment:
In view of your refactoring of the declarations for `A B C`, do we need so many `Levels` declared as well? Or would `a p ℓ : Level` suffice?

Reply:
Yup, nice spot!